x-pack/plugins/security_solution/server/endpoint/routes/actions/audit_log.ts

```diff
@@ -25,6 +26,10 @@ export function registerActionAuditLogRoutes(
       validate: EndpointActionLogRequestSchema,
       options: { authRequired: true, tags: ['access:securitySolution'] },
     },
-    actionsLogRequestHandler(endpointContext)
+    withEndpointAuthz(
+      { all: ['canIsolateHost'] },
```

Comment:
Just to clarify: we want to restrict log viewing that much?

Reply:
Initially, I did not have it this way, but during testing, I found that if the user does not have access to fleet indexes or our own `.*` indexes, then the API errors out anyway. So I went with checking that the user can isolate in order to see logs.